_posts/2020-01-07-jump_dev_2020.md

```diff
@@ -16,31 +16,22 @@ with interests in the software aspects of [JuMP](https://github.com/JuliaOpt/JuM
 related packages. The workshop is open for anybody to attend. In particular, we invite new
 contributors and those who have not met the core development team.
 
+## COVID-19
+
+At present, we are still planning for JuMP-dev to proceed as usual. However, we
+are aware that some institutions have placed travel bans until mid- to late-May.
```

Comment:
Suggested replace with: However, we are aware of the rapidly changing situation with the novel coronavirus outbreak and resulting institutional restrictions on international travel.